In-house patch to remove unnecessary dependencies from Ironic's
requirements files. The specific reasons are as follows:

kombu		Not applicable

lxml		Not applicable

websockify	Not applicable

oslo.rootwrap	Not applicable to Solaris

--- ironic-2014.2.1/ironic.egg-info/requires.txt.~1~	2015-02-19 10:50:54.000000000 -0800
+++ ironic-2014.2.1/ironic.egg-info/requires.txt	2015-05-12 13:36:29.313651947 -0700
@@ -4,9 +4,7 @@ alembic>=0.6.4
 anyjson>=0.3.3
 argparse
 eventlet>=0.15.1,<0.16.0
-kombu>=2.5.0
 lockfile>=0.8
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
 sqlalchemy-migrate==0.9.1
@@ -19,10 +17,8 @@ python-keystoneclient>=0.10.0
 python-swiftclient>=2.2.0
 stevedore>=1.0.0  # Apache-2.0
 pysendfile==2.0.0
-websockify>=0.6.0,<0.7
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
-oslo.rootwrap>=1.3.0
 oslo.i18n>=1.0.0  # Apache-2.0
 oslo.utils>=1.0.0                       # Apache-2.0
 pecan>=0.5.0
--- ironic-2014.2.1/requirements.txt.~1~	2015-02-19 10:45:47.000000000 -0800
+++ ironic-2014.2.1/requirements.txt	2015-05-12 13:37:03.904654352 -0700
@@ -7,9 +7,7 @@ alembic>=0.6.4
 anyjson>=0.3.3
 argparse
 eventlet>=0.15.1,<0.16.0
-kombu>=2.5.0
 lockfile>=0.8
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
 sqlalchemy-migrate==0.9.1
@@ -22,10 +20,8 @@ python-keystoneclient>=0.10.0
 python-swiftclient>=2.2.0
 stevedore>=1.0.0  # Apache-2.0
 pysendfile==2.0.0
-websockify>=0.6.0,<0.7
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
-oslo.rootwrap>=1.3.0
 oslo.i18n>=1.0.0  # Apache-2.0
 oslo.utils>=1.0.0                       # Apache-2.0
 pecan>=0.5.0
